Felienne/hedy
# Implement landing page

Provide a general summary of your changes in the Title of the PR

**Description**
Currently when a user creates an account they receive an e-mail to verify this account. After clicking this link they automatically re-direct to the main page. In this PR we implement a landing page to which they are re-directing, containing some general information on the possibilities and features of the Hedy website.

**Fix for**
This PR fixed #722 and fixed #981.

**How to test**
Create a new account and click the verification link in the mail. Notice the landing page and the instructions on the different functionalities.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
